storage/storage_helpers.cpp

```diff
@@ -33,7 +33,9 @@ bool IsEnoughSpaceForDownload(TCountryId const & countryId, Storage const & stor
 {
   NodeAttrs nodeAttrs;
   storage.GetNodeAttrs(countryId, nodeAttrs);
-  return IsEnoughSpaceForDownload(nodeAttrs.m_mwmSize - nodeAttrs.m_localMwmSize);
+  return IsEnoughSpaceForDownload(nodeAttrs.m_mwmSize > nodeAttrs.m_localMwmSize
```

Comment:
Yes. For the time being (21.04.2017) max mwm size is 105MB. It comes to mwm without osrm sections. I'm updating the constant in this PR.